invenio_circulation/transitions/transitions.py

```diff
@@ -151,6 +201,19 @@ def inner(self, loan, **kwargs):
                 )
                 if available_item_pid:
                     kwargs['item_pid'] = available_item_pid
+                    # if no pickup location was specified in the request,
+                    # assign a default one
+                    if not kwargs.get("pickup_location_pid"):
```

Comment:
I think you can remove this, it is covered by the if below

Reply:
Or move the below if inside this one, no? @ntarocco ?